Remotion/Web/UnitTests/Core/UI/Controls/Rendering/RendererBaseTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using JetBrains.Annotations;
+using Moq;
+using NUnit.Framework;
+using Remotion.Globalization;
+using Remotion.Web.UI.Controls;
+using Remotion.Web.UI.Controls.Rendering;
+
+namespace Remotion.Web.UnitTests.Core.UI.Controls.Rendering
+{
+  [TestFixture]
+  public class RendererBaseTest : RendererTestBase
+  {
+    [Test]
+    public void GetResourceManager_DoesNotKeepResourceManagerAlive ()
```

Comment:
Hmm... I think we should also add a test to show the same instance is returned twice for the same key while it's alive and a new instance is returned if the key's expired. Should have seen then earlier. sorry!

Reply:
I can't assert that a collected instance is different from a new one directly, I guess the existing test + a new one showing that the same instance is returned for the same keys is enough?

Reply:
> I can't assert that a collected instance is different from a new one directly,
True. 

> I guess the existing test + a new one showing that the same instance is returned for the same keys is enough?
+1